scripts/README.md

```diff
@@ -0,0 +1,38 @@
+# Leasing Scripts:
+
+
+Dynamic Free Pool:
+
+This script is used for reclaiming the non persistent nodes from projects in HaaS.
+
+Limitations:
+
+As of now, scripts location etc. are placed in my home folder. I want to change them to a service account(user which is not on my name). Also, this is more or less like a ticketing system which runs from a cron and executes the current script. So, there are chances of drifting if node gets just registered before  
```

Comment:
Hard-wrap lines in the docs to 72-79 chars (we haven't agreed upon an exact value for the docs, 79 is the rule for source code). This can get annoying to deal with in a text editor.

This goes for everything; this is just the most obvious case.